src/ICS20Transfer.sol

```diff
@@ -200,28 +226,28 @@ contract ICS20Transfer is IIBCApp, IICS20Transfer, IICS20Errors, Ownable, Reentr
     {
         bytes memory denomBz = bytes(packetData.denom);
         bytes memory sourceDenomPrefix = ICS20Lib.getDenomPrefix(packet.sourcePort, packet.sourceChannel);
-        bool originatorChainIsSource = !ICS20Lib.hasPrefix(denomBz, sourceDenomPrefix);
+        bool senderChainIsSource = !ICS20Lib.hasPrefix(denomBz, sourceDenomPrefix);
```

Comment:
I did a bit of renaming on this function and also the one below `getReceiveERC20AddressAndSource`.